ethereum/contracts/BasicInboundChannel.sol

```diff
@@ -28,25 +32,6 @@ contract BasicInboundChannel is InboundChannel {
         view
         returns (bool success)
     {
-        // Prove we can get the MMRLeaf that is claimed to contain our Parachain Block Header
```

Comment:
these comments are still important as they are changes we need to make - plz keep them in

Reply:
Sure ok. But in future, lets rather track this stuff via tickets or git branches. No reason to have commented-out code in main when people can track their in-progress work in side branches.

Reply:
agree in general but in this case this is security-critical code that is essential for us to get done and it is a good thing for it to be messing all over the place and drawing attention/disrupting clean code :)